governance/polls/Ratification Poll - Oracles Core Unit  Subproposal Set,  ORA-001 - June 14, 2021.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Ratification Poll for Oracles Core Unit (ORA-001) Subproposal Set - June 14, 2021
+summary: Signal your support or opposition for these proposals being accepted and implemented in the Maker Protocol. 
+discussion_link: https://forum.makerdao.com/t/mip39c2-sp12-adding-collateral-onboarding-core-unit/8037
+vote_type: Plurality Voting
+categories:
+   - MIPs
+options:
+   0: Abstain
+   1: Yes
+   2: No
+---
+# Ratification Poll for Oracles Core Unit (ORA-001) Subproposal Set - June 14, 2021
+
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, June 14 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to these proposals being accepted and implemented in the Maker Protocol.
+
+This Inclusion Poll MIP Set contains the following MIPs:
+
+* [MIP39c2-SP13: Adding Oracles Core Unit](https://github.com/makerdao/mips/blob/1a462c77d30bc603131b4f3c54a7486801395a7e/MIP39/MIP39c2-Subproposals/MIP39c2-SP3.md) | [Discussion Link](https://forum.makerdao.com/t/mip39c2-sp12-adding-collateral-onboarding-core-unit/8037)
+* [MIP40c3-SP15: Modify Oracles Core Unit Budget](https://github.com/makerdao/mips/blob/1a462c77d30bc603131b4f3c54a7486801395a7e/MIP39/MIP39c2-Subproposals/MIP39c2-SP3.md) | [Discussion Link](https://forum.makerdao.com/t/mip40c3-sp15-modify-oracles-core-unit-budget/8047)
+* [MIP41c4-SP13: Facilitator Onboarding, Oracles Core Unit](https://github.com/makerdao/mips/blob/1a462c77d30bc603131b4f3c54a7486801395a7e/MIP39/MIP39c2-Subproposals/MIP39c2-SP3.md) | [Discussion Link](https://forum.makerdao.com/t/mip41c4-sp13-facilitator-onboarding-oracles-core-unit/8046)
+
+The sentence summary and paragraph summary have been provided by the MIP author:
+
+**Sentence Summary**
+
+*MIP39c2-SP13 adds the Oracles Core Unit.*
+
+**Paragraph Summary**
+
+*($paragraph_summary)*
```

Comment:
I'd actually delete the missing paragraph summary if we don't hear back from Nik, and instead include the three sentence summaries:

The sentence summaries have been provided by the MIP author:

**Sentence Summary**

*MIP39c2-SP13 adds the Oracles Core Unit.*

*MIP40c3-SP15 adds the budget for the Oracles Core Unit.*

*MIP41c4-SP13 onboards Niklas Kunkel as Facilitator for the Oracle Core Unit.*